src/CLIFramework/IO/NullStty.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace CLIFramework\IO;
+
+class NullStty implements Stty
```

Comment:
what's the purpose of NullStty? is it only for testing? if so, name it with MockStty ?